test/e2e/networking.go

```diff
@@ -101,157 +100,225 @@ var _ = Describe("Networking", func() {
 		}
 	})
 
-	//Now we can proceed with the test.
-	It("should function for intra-pod communication", func() {
-		// TODO: support DNS on vagrant #3580
-		SkipIfProviderIs("vagrant")
+	//1 service and 120 seconds is the original networking.go test,
+	//Adding another two elements (i.e. 5, 300) to the arrays will
+	//create a second test, which creates 5 parallel services, and
+	//each on a different port, and waits for all 5 to complete the
+	//pinging.
+	serviceSoakTests := []struct {
+		service int
+		timeout float64
+	}{
```

Comment:
There are separate tests - each test is in an It clause below.   That way if a parallel services might fail but the simple one passes.